tests/cli/test_cli.py

```diff
@@ -30,6 +31,8 @@ def test_data_convert_help(run: Callable[..., RunResult]):
         assert output.outlines[i] == line
 
 
+@pytest.mark.linux
+@pytest.mark.darwin
```

Comment:
we could also leave this out and add an `xfail` on windows:
https://docs.pytest.org/en/latest/skipping.html#condition-parameter